app/views/admin/shared/_search_results.html.erb

```diff
@@ -0,0 +1,45 @@
+<p class="govuk-heading-s govuk-!-margin-bottom-3"><%= pluralize(number_with_delimiter(paginator.total_count), "document") %></p>
```

Comment:
Just wondering if it is strictly necessary to move this into a shared directory as its only the featurable_editions component that calls this partial?

Reply:
Yea i'm not sure exactly where this would live as it's called from multiple views by multiple controllers. Perhaps  `featurable_editions/search_results` 🤔